packages/ui/src/components/TimelineItem/TimelineItem.tsx

```diff
@@ -10,8 +10,8 @@ export interface TimelineItemProps extends FlexProps {
 }
 
 export const TimelineItem: React.FC<TimelineItemProps> = ({ step, date, information, ...props }) => {
-  const boxPadding = useBreakpointValue({ base: '18px', md: '32px' } as const) ?? '18px';
-  const dateVerticalMargin = useBreakpointValue({ base: '16px', md: '24px' } as const) ?? '16px';
+  const boxPadding = useBreakpointValue({ base: '18px', md: '32px' } as const, '18px');
```

Comment:
❓ Does the default value work? As I understand the intellisense description it should be `base` instead of `18px`
<img width="533" alt="CleanShot 2021-08-04 at 22 53 44@2x" src="https://user-images.githubusercontent.com/22938071/128253489-9b5814f9-7c71-42ce-9c48-7b5418806c8b.png">

Reply:
Right, corrected